test/bun.js/buffer.test.js

```diff
@@ -161,6 +161,17 @@ it("Buffer.from", () => {
   gc();
 });
 
+it("Buffer.from latin1 vs ascii", () => {
```

Comment:
Thank you for writing a test!!

Reply:
BTW can you tell me why you put `gc()` here and there in your tests?

Reply:
gc issues are reproduce inconsistently

Running the GC manually like this helps catch a lot of bugs